src/resolvers/Query.test.ts

```diff
@@ -0,0 +1,194 @@
+import User from '../models/user.model'
+import Query from './Query'
+
+jest.mock('../models/user.model')
+
+describe('User suit', () => {
+
+  beforeEach(() => {
+  })
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  describe('Find an user', () => {
+    it('Find user by Id successfully', async () => {
+
+      //GIVEN
+      const userExpected = new User('name',
+        new Date(2007, 2, 3),
+        'Tv 4 # 54-56',
+        'test user',
+        'www.testuser.com')
+      userExpected.id = '12345'
+
+      //WHEN
+      User.find = jest.fn(() => Promise.resolve(userExpected))
+
+      //THEN
+      const userFound = await Query.findUser(null, {
+        id: '12345'
+      }, null, null)
+      expect(userFound).not.toBe(null)
+      expect(userFound).toBe(userExpected)
+    })
+    it('Find user by Name successfully', async () => {
+
+      //GIVEN
+      const userExpected = new User('name',
+        new Date(2007, 2, 3),
+        'Tv 4 # 54-56',
+        'test user',
+        'www.testuser.com')
+      userExpected.id = '12345'
+
+      const responseExpected = {
+        lastEvaluatedUser: { id: userExpected.id, name: userExpected.name },
+        users: userExpected,
+      }
+
+      //WHEN
+      User.findByName = jest.fn((name: String) => Promise.resolve(responseExpected))
+
+      //THEN
+      const userFound = await Query.findUsersByName(null, {
+        data: {
+          name: 'name'
+        }
+      }, null, null)
+      expect(userFound).not.toBe(null)
+      expect(userFound).toBe(responseExpected)
+    })
+  })
+  describe('Find all user', () => {
+    it('Get first 2 users successfully', async () => {
+      //GIVEN
+      const usersExpected = [new User(
+        'name',
+        new Date(2007, 2, 3),
+        'Tv 4 # 54-56',
+        'test user',
+        'www.testuser.com'
+      ), new User(
+        'name2',
+        new Date(2007, 2, 3),
+        'Tv 42 # 54-56',
+        'test user2',
+        'www.testuser2.com'
+      )]
+
+      usersExpected[0].id = '12345'
+      usersExpected[1].id = '54321'
+
+      const responseExpected = {
+        lastEvaluatedId: usersExpected[1].id,
+        users: usersExpected,
+      }
+
+      //WHEN
+      User.getAll = jest.fn(() => Promise.resolve(responseExpected))
+
+      //THEN
+      const usersFound = await Query.getUsers(null, {}, null, null)
+      expect(usersFound).not.toBe(null)
+      expect(usersFound).toBe(responseExpected)
+    })
+  })
+  describe('Get address coordinates', () => {
+    it('Get address coordinates successfully', async () => {
+      //GIVEN       
+      const jsonExpected = `{
```

Comment:
Why not extract this big JSON to another module or a file of its own? :)

Reply:
You right, it would be a pending refactor. This was another example of code that I put it there for testing and forget to refactor it.